File: apzkr-pzpi-21-2-butsulin-ihor/Task4-MobileClient/APZMobile/app/src/main/java/com/osigram/apz2024/mobile/models/Task.kt
package com.osigram.apz2024.mobile.models

import kotlinx.serialization.Serializable

@Serializable
data class Task(val ID: ULong, val workerID: ULong, val worker: Worker, val fromSlotID: ULong, val fromSlot: Slot, val toSlotID: ULong, val toSlot: Slot, val status: Boolean)

File: apzkr-pzpi-21-2-butsulin-ihor/Task1-Server/apz-backend/types/models/Task.go
package models

import "gorm.io/gorm"

type Task struct {
	gorm.Model
	WorkerID   uint   `json:"workerID"`
	Worker     Worker `json:"worker"`
	FromSlotID uint   `json:"fromSlotID"`
	FromSlot   Slot   `json:"fromSlot"`
	ToSlotID   uint   `json:"toSlotID"`
	ToSlot     Slot   `json:"toSlot"`
	Status     bool   `json:"status"`
}
